src/neural-network.js

```diff
@@ -22,7 +22,7 @@ export default class NeuralNetwork {
       learningRate: 0.3,
       callback: null,
       callbackPeriod: 10,
-      reinforce: false
+      trainTimeMs: -Infinity
```

Comment:
Can the `trainTimeMs` be put into a seperate pr?  Want to keep the pr's as lean as possible.